### PR TITLE
feat: add sample and test for getting an access token from an imperso…

### PR DESCRIPTION
**Pull request by irataxy**
…nated SA

Kindly help set up a similar CL: 572924785 testing resource.

Fixes b:302748677

- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

**Comment by pattishin** [comment]
Hi @irataxy! Do you mind taking a gander at the auth test failure? Happy to provide relevant logs if more information is needed.

**Comment by irataxy** [comment]
> Hi @irataxy! Do you mind taking a gander at the auth test failure? Happy to provide relevant logs if more information is needed.

Kindly help set up a similar CL: 572924785 testing resource.

File: auth/system-test/accessTokenFromImpersonatedCredentials.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+'use strict';
+
+const assert = require('assert');
+const path = require('path');
+const {execSync} = require('child_process');
+
+const cwd = path.join(__dirname, '..');
+const cmd = 'node accessTokenFromImpersonatedCredentials.js';
+
+const impersonatedServiceAccount = 'name@project.iam.gserviceaccount.com';
```
Comment (irataxy) [inline auth/system-test/accessTokenFromImpersonatedCredentials.test.js:24]: @alclee-google 
Needs to be changed to the service account name set up by the reviewer

Comment (pattishin) [reply auth/system-test/accessTokenFromImpersonatedCredentials.test.js:24]:
```suggestion
const impersonatedServiceAccount = 'auth-samples-testing@long-door-651.iam.gserviceaccount.com';
```

**Review by Sita04 — APPROVED** [review]

**Pull request merged**